docs/static/running-logstash-windows.asciidoc

```diff
@@ -1,3 +1,147 @@
+PS C:\Windows\system32> SETX /m JAVA_HOME "C:\Program Files\Java\jdk-11.0.3"
+PS C:\Windows\system32> SETX /m PATH "$env:PATH;C:\Program Files\Java\jdk-11.0.3\bin;"
+-----
+**** Exit PowerShell, then open a new PowerShell session and run `Write-Host $env:JAVA_HOME` to verify
+
+===== `Java -version`
+** This command produces output similar to the following:
++
+[source,txt]
+-----
+PS C:\> Java -version
+java version "11.0.3" 2019-04-16 LTS
+Java(TM) SE Runtime Environment 18.9 (build 11.0.3+12-LTS)
+Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.3+12-LTS, mixed mode)
+-----
+
+NOTE: As of the publication of this document, please review this https://github.com/elastic/logstash/issues/10496[known issue that impacts Java 11] before proceeding.
+
+Once you have <<setup-logstash>> and validated JVM pre-requisites, you may proceed.  
+
+NOTE: For the examples listed below, we are running Windows Server 2016, Java 11.0.3, have extracted the https://www.elastic.co/downloads/logstash[Logstash ZIP package] to `C:\logstash-7.1.0\`, and using the example `syslog.conf` file shown below (stored in `C:\logstash-7.1.0\config\`).
+
+[[running-logstash-windows-manual]]
+==== Running Logstash manually
+Logstash can be run manually using https://docs.microsoft.com/en-us/powershell/[PowerShell].  Open an Administrative https://docs.microsoft.com/en-us/powershell/[PowerShell] session, then run the following commands:
++
+[source,txt]
+-----
+PS C:\Windows\system32> cd C:\logstash-7.1.0\
+PS C:\logstash-7.1.0> .\bin\logstash.bat -f .\config\syslog.conf
+-----
+
+Wait for the following messages to appear, to confirm Logstash has started successfully:
++
+[source,txt]
+-----
+[logstash.runner          ] Starting Logstash {"logstash.version"=>"7.1.0"}
+[logstash.inputs.udp      ] Starting UDP listener {:address=>"0.0.0.0:514"}
+[logstash.agent           ] Successfully started Logstash API endpoint {:port=>9600}
+-----
+NOTE: Logstash can be stopped by sending the `ctrl` + `c` keys in the PowerShell console.
+
+[[running-logstash-windows-nssm]]
+==== Running Logstash as a service with NSSM
+NOTE: It is recommended to validate your configuration works by running Logstash manually before you proceed.
+
+Download https://nssm.cc/[NSSM], then extract `nssm.exe` from `nssm-<version.number>\win64\nssm.exe` to `C:\logstash-7.1.0\bin\`.  Then open an Administrative https://docs.microsoft.com/en-us/powershell/[PowerShell] session, then run the following commands:
++
+[source,txt]
+-----
+PS C:\Windows\system32> cd C:\logstash-7.1.0\
+PS C:\logstash-7.1.0> .\bin\nssm.exe install logstash
+-----
+
+Once the `NSSM service installer` window appears, specify the following parameters in the `Application` tab:
+
+** In the `Application` tab:
+*** Path: Path to `logstash.bat`: `C:\logstash-7.1.0\bin\logstash.bat`
+*** Startup Directory: Path to the `bin` directory: `C:\logstash-7.1.0\bin`
+*** Arguments: For our example to start Logstash: `-f C:\logstash-7.1.0\config\syslog.conf`
+
```

Comment:
Rob, I would think about adding a Note for leaving arguments blank if they want to use `pipelines.yml`.
NOTE: Leave arguments blank if you are using `pipelines.yml`. See example below.

Reply:
@triggerman86, I've added a note for multiple pipeline support in [a3a75fe](https://github.com/elastic/logstash/commit/a3a75fe84300a0b39741ba1cf308dfb006a13db4)

Reply:
@triggerman86, thank you for the feedback!  You bring up a great point that this may cause more issues, especially if arguments are used.  I've made another update to clarify that in a production environment, we recommend using the logstash settings file to control Logstash execution (i.e., no `-f .\config\syslog.conf` arguments).  I added that as a note in each place there was an example of an argument being used.